source/apps/export.txt

```diff
@@ -0,0 +1,210 @@
+            directory. You can configure the command to create a new
+            directory instead by specifying the directory path with the
+            ``--local`` flag.
+
+            .. code-block:: shell
+
+               realm-cli pull --remote=<App ID> --local=<Output Directory Path>
+
+            .. tip::
+
+               The directory specified by ``--local`` must not already
+               exist, otherwise the CLI throws an error and does not
+               pull the configuration files.
+
+            The command also supports :ref:`additional flags
+            <realm-cli-pull>` that you can optionally include to
+            customize your app. The following table lists common flags
+            you might use:
+
+            .. list-table::
+
+               * - ``--include-dependencies``
+                 - If specified, external dependencies are exported as a
+                   ``node_modules`` archive in the ``/functions``
+                   directory.
+
+               * - ``--include-hosting``
+                 - If specified, hosted static assets are exported
+                   in the ``/hosting/files`` directory.
+
+   .. tab::
+      :tabid: api
+
+      .. procedure::
+
+         .. step:: Authenticate a MongoDB Atlas User
+
+            .. include:: /includes/api-authenticate-instructions.rst
+
+         .. step:: Get the Atlas Project ID
+
+            To export an App, you need the Project ID (sometimes referred to as
+            the Group ID) of the Atlas project that contains the app.
+
+            To learn how to find your Project ID, see :ref:`find-atlas-project-id`.
+
+         .. step:: Get the App ID
+
+            .. include:: /includes/api-get-application-id.rst
+
+         .. step:: Export Your App
+
+            To export your App, send a request to the
+            :admin-api-endpoint:`Export an App
+            <operation/adminExportApplication>` endpoint.
+
+            Your request should include:
+
+            - The ``access_token`` from step 1 in a :mdn:`Bearer token
+              <Web/HTTP/Authentication>` ``Authorization`` header.
+
+            - The :admin-api-endpoint:`Group ID/Project ID
+              <section/Project-and-Application-IDs>` from step 2.
+
+            - The :admin-api-endpoint:`Application ID
+              <section/Get-Authentication-Tokens>` from step 3.
+
+            .. code-block:: shell
+
+               curl -X GET \
+                 https://realm.mongodb.com/api/admin/v3.0/groups/{groupId}/apps/{appId}/export \
+                 -H 'Authorization: Bearer <access_token>' \
+                 -H 'Content-Type: application/json' \
+                 --output ./exported-app.zip
+
+            If the request is successful, the endpoint returns a ``200``
+            response that contains a ``.zip`` directory of your
+            application's current configuration files.
+
+            You can export the configuration for a specific deployment
```

Comment:
seems like this should be above the 'If the request was successful...' verbiage, since it's giving additional options on what your request can include